src/otk/command.py

```diff
@@ -48,27 +48,18 @@ def root() -> int:
 
 
 def _process(arguments: argparse.Namespace, dry_run: bool) -> int:
-    src = pathlib.Path("/dev/stdin" if arguments.input is None else arguments.input)
-    dst = (
-        pathlib.Path("/dev/stdout" if arguments.output is None else arguments.output)
-        if not dry_run
-        else None
-    )
-
-    if not src.exists():
-        log.fatal("INPUT path %r does not exist", str(src))
-        return 1
-
-    if not dry_run and not dst.parent.exists():
-        log.fatal("OUTPUT path %r does not exist", str(dst))
-        return 1
+    src = sys.stdin if arguments.input is None else open(arguments.input)
+    if not dry_run:
+        dst = sys.stdout if arguments.output is None else open(arguments.output, "w")
 
     # the working directory is either the current directory for stdin or the
     # directory the omnifest is located in
-    cwd = pathlib.Path.cwd() if arguments.input is None else src.parent
+    cwd = (
```

Comment:
I really don't get why `ruff` puts this into a multi-line construct with a `(` around but above for `dst = ` it does not do that. I'm really not a fan :(

Reply:
It's because this line is > 88 chars while `dst` is not.
we could add a `.ruff.toml` with
```
line-length = 120
```

Reply:
@supakeen any preference for line lengths in this project? I think nowadays with those widescreen, high-res. monitors 120 chars seem fine for me

Reply:
Fwiw, I was a bit grumpy when I wrote it - it's okay if we keep it as is, I'm just not a fan :) [as an additonal datapoint, osbuild uses 120 for the line length]

Reply:
I have no strong opinions; 120 is consistent with `osbuild` so let's do that.

Reply:
Thank you! I opened https://github.com/osbuild/otk/pull/108